yapapi/storage/gftp.py

```diff
@@ -245,8 +251,30 @@ def read_use_gftp_close_env_var() -> Optional[bool]:
         return None
 
 
+MIN_GFTP_VERSION_THAT_CAN_GFTP_CLOSE = semantic_version.Version("0.7.3")
```

Comment:
I know _naming is hard_, but can we shorten this a bit? `MIN_CLOSE_URLS_GFTP_VERSION` perhaps?